src/client/components/Modal/Modal.component.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-param-reassign */
+import React from 'react';
+import './Modal.styles.css';
+import PropTypes from 'prop-types';
+import Button from '../Button/Button.component';
+import addedToCart from '../../assets/images/added_to_cart.png';
+
+export default function Modal({
+  modalIsOpen,
+  message,
+  cornerCloseButton,
+  primaryButtonLabel,
+  showConfirmIcon,
+  buttonFunction,
+  secondaryButtonFunction,
+  secondaryButtonLabel,
+}) {
+  return (
+    <div className="modal-background">
+      <div className="modal-box">
+        {cornerCloseButton && (
+          <button
+            onClick={() => modalIsOpen(false)}
+            type="button"
+            className="closing-modal-button"
+          >
+            X
+          </button>
+        )}
+        <p className="modal-text">{message}</p>
+        {showConfirmIcon && (
```

Comment:
Maybe it would make more sense to allow the user to pass in the content to display in the modal? 
That way you can use the modal for a lot of different things.

Reply:
can you elaborate on this, do you want users to be able to decide exactly which html tags are put into the modal, and their order?

Reply:
no the modal shouldnt be flexible for user according to wireframes. You can check the link below: https://www.figma.com/file/yo4QT2EcPo83HAMI3ydMxb/HYF-Wireframes?node-id=4%3A1324

Reply:
In general, I wouldn't assume such details from the wireframe alone. 
What if there is another wireframe that also uses a modal but that modal looks different? Should that be another specific component or should this modal be re-used? 
It depends.

But anyway, if your decision is that this modal should be more specific then I think it would be better to write the code that way, make the code less flexible and more clear that it's the modal for the product confirmation.